common/djangoapps/student/models.py

```diff
@@ -2024,3 +2043,63 @@ def get_enrollment_attributes(cls, enrollment):
             }
             for attribute in cls.objects.filter(enrollment=enrollment)
         ]
+
+    @classmethod
+    def get_enrollment_attribute(cls, enrollment, **kwargs):
+        """Retrieve a single enrollment attribute.
+
+        Args:
+            enrollment(CourseEnrollment): 'CourseEnrollment' for which list is to retrieve
+            kwargs: Any additional parameters to scope the query
+
+        Returns: dict
+
+        Example:
+        >>> CourseEnrollmentAttribute.get_enrollment_attribute(CourseEnrollment, namespace='credit', name='provider_id')
+        {
+            "namespace": "credit",
+            "name": "provider_id",
+            "value": "hogwarts",
+        }
+        """
+        try:
+            attribute = cls.objects.get(enrollment=enrollment, **kwargs)
+        except ObjectDoesNotExist:
+            return {}
+
+        return {
+            "namespace": attribute.namespace,
+            "name": attribute.name,
+            "value": attribute.value,
+        }
+
+
+class EnrollmentRefundConfiguration(ConfigurationModel):
+    """
+    Configuration for course enrollment refunds.
+    """
+
+    # TODO: Django 1.8 introduces a DurationField
+    # (https://docs.djangoproject.com/en/1.8/ref/models/fields/#durationfield)
+    # for storing timedeltas which uses MySQL's bigint for backing
+    # storage. After we've completed the Django upgrade we should be
+    # able to replace this field with a DurationField named
+    # `refund_window` without having to run a migration or change
+    # other code.
+    refund_window_microseconds = models.BigIntegerField(
```

Comment:
I definitely agree that the granularity is excessive, but I want to ensure that we can smoothly change this to a `DurationField` after the Django 1.8 upgrade. I'd rather use the officially-supported field types when we can, rather than rolling our own.